docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -123,9 +128,128 @@ outbreaks due to lack of vaccination in small communities.
 Data Description
 ----------------
 
-.. todo::
+.. list-table:: Definitions
+   :widths: 25 50
+   :header-rows: 1
+
+   * - State
+     - Definition
```

Comment:
@kiranklc @beatrixkh @XiaoluQianUW On Friday we discussed adding a "State Name" column to this table. Do we still agree that that's a good idea?

Reply:
It might seem a bit redundant for simple SIR or SIS models, but I think it becomes more important to come up with short descriptive names in models with less standard state definitions, like neural tube defects, neonatal hemolytic disease, or the tuberculosis model.

Reply:
I agree that's a good idea. I will add the column to the definitions table.